tests/test-http-client-transport/src/test/resources/jetty-logging.properties

```diff
@@ -2,6 +2,6 @@ org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
 #org.eclipse.jetty.LEVEL=DEBUG
 #org.eclipse.jetty.client.LEVEL=DEBUG
 #org.eclipse.jetty.http2.LEVEL=DEBUG
-#org.eclipse.jetty.http2.hpack.LEVEL=INFO
+org.eclipse.jetty.http2.hpack.LEVEL=INFO
```

Comment:
Was this a temporary change for debugging?

Reply:
No, it's a permanent change since you rarely want hpack logging outside the hpack module.